ota-plus-web/app/reactapp/src/config.js

```diff
@@ -243,6 +243,12 @@ export const assets = {
 export const HELP_ICON_DARK = '/assets/img/new-app/24/help-24-dark.svg';
 export const LINK_BUTTON_ICON = '/assets/img/icons/link_button.svg';
 export const SIGN_OUT_ICON = '/assets/img/new-app/24/sign-out.svg';
+export const CAMPAIGNS_ICON = '/assets/img/new-app/24/campaigns-active-solid-24.svg';
+export const DEVICE_ICON = '/assets/img/new-app/24/devices-active-outline-24.svg';
+export const GROUP_ICON = '/assets/img/new-app/24/device-group-active-outline-24.svg';
+export const SOFTWARE_ICON = '/assets/img/new-app/24/software-versions-active-outline-24.svg';
+export const UPDATE_ICON = '/assets/img/new-app/24/software-updates-active-outline-24.svg';
+export const WARNING_LUI_ICON = '/assets/img/new-app/16/LUI-icon-pd-attention-solid-16-1.svg';
```

Comment:
Do we have another warning icons than lui?

Reply:
I don't think so, but I thought it is a good idea to somehow mark icons used throughout LUI compliant designs.

Reply:
WARNING_ICON will be enough I think.